app/views/playbook/_nav_top.html.slim

```diff
@@ -0,0 +1,16 @@
+.clearfix
+  .sm-col.sm-col-5
+    = link_to \
+      image_tag(asset_url('logo.svg'),
+        alt: APP_NAME,
+        class: 'align-bottom',
+        width: 151), root_path, class: 'inline-block'
+    h1.mb2.h3.red.caps.ls-5 IDENTITY PLAYBOOK
+  .sm-col.sm-col-7.sm-right-align
+    .sm-show.pb-nav--top
```

Comment:
Not sure what the mobile view is going to look like quite yet, so just temporarily hiding it until its styled properly.